stepactions/get-unreleased-bundle/0.1/README.MD

```diff
@@ -0,0 +1,25 @@
+# get-unreleased-bundle stepaction
+
+This StepAction retrieves the highest bundle version from a specified package and channel in the provided FBC fragment. 
+If package name and/or channel name are not specified, default values are used.
+Default package name determination:
+    
+    * If there is only one 'olm.package', it's name is returned
+    
+    * If there are multiple 'olm.package' entries, the package with the highest bundle version in its 'defaultChannel' is selected
+Default channel name determination:
+    
+    * The default channel name corresponds to the 'defaultChannel' entry of the selected package
+The StepAction checks whether the bundle is unreleased, specifically, not present in the Red Hat production Index Image (registry.redhat.io/redhat/redhat-operator-index). 
+If the bundle is unreleased, the StepAction returns it.
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|fbcFragment|The FBC fragment image.||true|
+|packageName|The OLM package name.|If there is only one 'olm.package', it's name is returned. If there are multiple 'olm.package' entries, the package with the highest bundle version in its 'defaultChannel' is selected|false|
```

Comment:
In the default value field I suggest `An OLM package name present in the fragment or leave it empty so the step will determine the default package name as described above`

Reply:
thanks for the suggestions! I pushed them in the latest commit.